### DISCUSSION TITLE
Generated header file from image png

### DISCUSSION THREAD

**Discussion by aullidolunar** [wxFormBuilder/wxFormBuilder#844] [Q&A]
Hi. Using wxFB Version 4.2.1-0-g80c4cb6. I'd like to know what tool wxFB uses to generate the header files from png images. With Meson Build System I would like to generated a header file like wxFB does (or is it do?) from a png image based on some criteria of the host system is my app beeing build. Thanks.

**Comment by sodevel** [comment]
I'm sorry but i don't know what you mean, i am not aware that wxFB does generate header files from PNG's, neither for the generated code nor for its own code.
